src/registrar/config/urls.py

```diff
@@ -0,0 +1,23 @@
+"""URL Configuration
+
+For more information see:
+    https://docs.djangoproject.com/en/4.0/topics/http/urls/
+"""
+
+from django.conf import settings
+from django.contrib import admin
+from django.urls import include, path
+
+from registrar.views import health
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
```

Comment:
Is this where django puts its admin interface?

Reply:
Yep.